.github/workflows/ci.yml

```diff
@@ -0,0 +1,74 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - main
+      - dev/*
```

Comment:
@sudomaze, do you think we should have a stable branch that deploy to pypi directoy on merge or main is enough? I think we should also maintain another testing branch for test.pypi. What do you think?

Reply:
The idea that I had in mind was to publish to `pypi` through GitHub Releases triggers, so we will have a GitHub Action of publishing only when something is released. This will limit any mistakes of publishing bad versions and this disconnects development code and stable releases.

We can add a nightly version of the package which pulls from a stable branch (i.e. `nightly-release` branch).

Reply:
I see! so only one main branch is enought.